### PR TITLE
Add more debug information around strange failure in e2e tests

### PR DESCRIPTION
**Pull request by mwielgus**
To better understand problems in https://github.com/kubernetes/kubernetes/issues/33754. 

cc: @fgrzadkowski @piosz

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/37481)

### CODE REVIEW

File: test/e2e/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -210,6 +210,34 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Slow]", func() {
...
+					newNode,
+					"--project="+framework.TestContext.CloudConfig.ProjectID,
+					"--zone="+framework.TestContext.CloudConfig.Zone).Output(); err != nil {
+					glog.Infof("Gcloud compute instances describe: %s", output)
+				} else {
+					glog.Errorf("Failed to get instances describe: %v", err)
+				}
+			}
+
+			// TODO: possibly remove broken node from newNodesSet to prevent removeLabel from craching.
```
Comment (fgrzadkowski) [inline test/e2e/cluster_size_autoscaling.go]: crashing

**Comment by fgrzadkowski** [comment]
LGTM.

**Comment by mwielgus** [comment]
@k8s-bot unit test this

**Comment by mwielgus** [comment]
Jenkins Kops AWS e2e succeeded (according to the details). The state was not reflected in PR checks.

**Comment by mwielgus** [comment]
@k8s-bot kops aws e2e test this

**Comment by dims** [comment]
@k8s-bot test this

**Comment by saad-ali** [comment]
Does this need to be cherry-picked to 1.5? If so make sure to add `cherrypick-candidate` label

**Comment by saad-ali** [comment]
> Does this need to be cherry-picked to 1.5? If so make sure to add cherrypick-candidate label

Ping

**Comment by saad-ali** [comment]
>> Does this need to be cherry-picked to 1.5? If so make sure to add cherrypick-candidate label
> Ping

Ping

**Comment by saad-ali** [comment]
>>> Does this need to be cherry-picked to 1.5? If so make sure to add cherrypick-candidate label
>> Ping
>
> Ping

Ping

**Pull request merged**